### ISSUE TITLE
Interacting with the refugees' cars can cause a mutiny

### ISSUE THREAD

**Issue by SariusSkelrets** [CleverRaven/Cataclysm-DDA#55575]
### Describe the bug

When interacting with the refugee camp car, you get a prompt mentionning consequences

Ignoring that prompt and interacting with the car in any way (even just getting inside) can make your followers want to kill you as if they were bandits

### Steps to reproduce

1. Debug get a NPC follower
2. Get to the refugee camp
3. Touch one of their cars
4. Notice how you loyal follower is now trying to kill you

### Expected behavior

Consequences coming from the owner faction and not from my loyal followers

### Screenshots

![cdda car 2](https://user-images.githubusercontent.com/68650913/155030954-6be6f3ae-02be-445b-8c62-9e22b1b1511f.png)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0 2009
- Game Version: 1d946db [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    [WIP] Mythos Mod - No Zombies [Mythos]
]

**Comment by Inglonias** [comment]
I feel like this is something I might do if I saw my leader steal shit like this. Or at least, that's what I would want to think.

**Comment by SariusSkelrets** [comment]
My concern isn’t about the reaction, but about how, except directly attacking them, touching these cars is literally the only way to piss them off. Even if they see you stealing everything you can take without someone noticing they won’t care at all

I could kill a guard in front of them, no reaction. I could butcher that guard in front of them, no reaction. I could eat the guard’s raw flesh right in their face, no reaction. I could cook that dead guard with their help and put that meat in their food stock, no reaction.

But if I touch a car that isn’t mine, I’m a monster that needs to be killed right now. I feel like they should be more loyal or at least understand that our group need a car over pleasing people that won’t give us anything

**Comment by LeahLuong** [comment]
I encountered this before & decided it was worth savescumming; agree it should be fixed. The follower does give some verbal warnings before going turncoat if you have some rapport but I thought it was silly they'd try to kill me over siphoning a couple of L's of diesel from a broken-down 'fugee hoopty. Wasn't even happy when I started to put it back. After savescumming, I parked them around the corner of the center where they wouldn't see me thieving. Worked great!

**Comment by Night-Pryanik** [comment]
Can't confirm on 0.F-12232.

I opened Free Merchants' bus while both their guard and my follower clearly saw me. I ignored prompt about consequences, and guard started attacking me, but my follower stayed loyal to me. No matter how many times I stepped into the bus, follower opinion about me didn't change, and he didn't want to attack me.

I'm closing this for now, if it's reproducible somehow, ping me to reopen.
